### PR TITLE
NGE-25: Add web component docs in storybook

### PR DESCRIPTION
**Pull request by AdrianRRo**

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -1,4 +1,10 @@
 // .storybook/preview.js
+import { setCustomElements } from '@storybook/web-components';
+
+// funcion que mezcle todos los json
+const counter = require('./custom-elements.json');
+
+setCustomElements(counter);
```
Comment (JackBlakeston) [inline .storybook/preview.js]: Do we need to hardcode all our web components here?

File: plugins/web-components-plugin.js [hunk 2]
```diff
@@ -19,21 +45,77 @@ function webComponentsPlugin() {
...
+
+        /*const dirPath = path.join(
+          '.storybook',
+          'components',
+          componentType,
+          `${componentName}.json`,
+        );
+
+        await writeFileWithDir(dirPath, output);
+        */
```
Comment (JackBlakeston) [inline plugins/web-components-plugin.js]: leftover comment?

**Review by JackBlakeston — APPROVED** [review]

File: .storybook/preview.js (hunk 1 above)
Comment (AdrianRRo) [reply .storybook/preview.js]: no, I just forgot to change the name of the variable xD that was the name I used in a previous test

File: plugins/web-components-plugin.js (hunk 2 above)
Comment (AdrianRRo) [reply plugins/web-components-plugin.js]: yes! good catch

**Pull request merged**